src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/ProvidersController.cs

```diff
@@ -117,6 +127,54 @@ public void Head(long ukprn)
             Get(ukprn);
         }
 
+        /// <summary>
+        /// Get list of active apprenticeships for a given provider
+        /// </summary>
+        /// <param name="ukprn">UKPRN</param>
+        /// <returns>A list of active apprenticeships sorted by name alphabetically, then type, then level</returns>
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<ProviderApprenticeship>))]
+        [SwaggerResponse(HttpStatusCode.InternalServerError)]
+        [SwaggerResponse(HttpStatusCode.BadRequest, BadUkprnMessage)]
+        [Route("providers/{ukprn}/active-apprenticeships", Name = "GetActiveApprenticeshipsByProvider")]
+        [ExceptionHandling]
+        public IEnumerable<ProviderApprenticeship> GetActiveApprenticeshipsByProvider(long ukprn)
+        {
+            if (ukprn.ToString().Length != 8)
+            {
+                throw HttpResponseFactory.RaiseException(HttpStatusCode.BadRequest, BadUkprnMessage);
+            }
+
+            var standards = _getProviders.GetStandardsByProviderUkprn(ukprn);
+
+            var apprenticeships = standards.Where(x => DateHelper.CheckEffectiveDates(x.EffectiveFrom, x.EffectiveTo))
+                .Select(standard => new ProviderApprenticeship
+                {
+                    Name = standard.Title,
+                    Level = standard.Level,
+                    Type = ApprenticeshipTrainingType.Standard.ToString(),
+                    TrainingType = ApprenticeshipTrainingType.Standard,
+                    Identifier = standard.StandardId.ToString()
+                })
+                .ToList();
+
+            var frameworks = _getProviders.GetFrameworksByProviderUkprn(ukprn);
+
+            apprenticeships.AddRange(frameworks.Where(x => _activeFrameworkChecker.CheckActiveFramework(x.FrameworkId, x.EffectiveFrom, x.EffectiveTo))
+                .Select(framework => new ProviderApprenticeship
+                {
+                    Name = framework.PathwayName,
+                    Level = framework.Level,
+                    Type = ApprenticeshipTrainingType.Framework.ToString(),
+                    TrainingType = ApprenticeshipTrainingType.Framework,
+                    Identifier = framework.FrameworkId
+                }));
+            var take = _applicationSettings.ProviderApprenticeshipsMaximum;
+
+            return apprenticeships.OrderBy(x => x.Name)
```

Comment:
Do we need a max?
The caller can maybe do take on that side?
Or we can add query parameter for the user to ask who many they want?